api/src/services/healthService.js

```diff
@@ -51,140 +46,83 @@ async function checkAccountsBalances(req, res) {
-      }
-    })
+  if (!cereNetworkService.initialized()) {
+    errors.push('Cere network service is not initialized yet');
   }
-}
-
-function validateNoEmptyFields(fields) {
-  for (var name in fields) {
-    if (!fields[name] || !fields[name].length) {
-      throw new Error(`Field "${name}" is empty`);
-    }
-  }  
-}
 
-function splitParams(params) {
-  return params && params.split(SEPARATOR)
-}
-
-async function liveness(req, res) {
-  res.status(200).json({
-    msg: 'Liveness probe is ok',
-  });
+  if (!ethNetworkService.initialized()) {
+    errors.push('Ethereum network service is not initialized yet');
+  }
+  
+  errors.length
+    ? res.status(503).json({msg: errors})
+    : res.status(200).json({msg: 'Readiness probe is ok'});
 }
 
-async function readiness(req, res) {
-  let errors = [];
-  
-    let client;
-    try {
-      client = await getClient();      
-    } catch (error) {
-      console.error(error);
-      errors.push(`Unable to connect to database:${err.message}`);
-    } finally {
-      client?.end();
-    }
-  
-    if (!cacheService.initialized()) {
-      errors.push('Cache service is not initialized yet');
-    }
+async function checkBlockchainHealth(req, res) {
+  const { query } = req;
+  let networks = splitParams(query.networks);
   
-    if (!cereNetworkService.initialized()) {
-      errors.push('Cere network service is not initialized yet');
+  if (networks) {
+    const validationErrors = validateNetworks(networks);
+    if (validationErrors.length) {
+      return res.status(400).json({msg: validationErrors });
     }
+  } else {
+    networks = Object.values(networkNames);
+  }
   
-    if (!ethNetworkService.initialized()) {
-      errors.push('Ethereum network service is not initialized yet');
-    }
-    
-    errors.length
-      ? res.status(503).json({msg: errors})
-      : res.status(200).json({msg: 'Readiness probe is ok'});
-}
 
-async function checkBlockchainHealth(req, res) {
-    const { query } = req;
-    const networks = splitParams(query.networks);
-    let errors = [];
-    
+  let errors = [];
+  await Promise.all(networks.map(async network => {
     try {
-      validateNoEmptyFields({networks});
-      validateNetworksNames(networks);
+      await cereNetworkService.getHealth(network);
     } catch (error) {
-      return res.status(400).json({
-        msg: error.message,
-      });
+      console.error(error);
+      errors.push(`Unable to connect to ${network} network`);
     }
+  }));
 
-    await Promise.all(networks.map(async network => {
-      try {
-        await cereNetworkService.getHealth(network);
-      } catch (error) {
-        console.error(error);
-        errors.push(`Unable to connect to ${network} network`);
-      }
-    }));
-
-    res.status(200).json(errors.length ? { errors } : { msg: `${networks.join(',')} status is ok`});
+  res.status(200).json(errors.length ? { errors } : { msg: `${networks.join(',')} status is ok`});
```

Comment:
msg -> "X failed"
errors -> list of errors

Reply:
fixed